### PR TITLE
test: sqlness test for insert default

### PR DESCRIPTION
**Pull request by waynexia**
Signed-off-by: Ruihang Xia <waynestxia@gmail.com>

I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Add sqlness tests for INSERT DEFAULT implemented in #854.

cc @lizhemingi please take a look!

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Review by evenyag — APPROVED** [review]

**Comment by lizhemingi** [comment]
Should we add a test like a column can not build a default value atomically(such as no default constraint TIME INDEX column) but use this default insert, and expect it will fail ?

File: tests/cases/standalone/insert/insert_default.result [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Affected Rows: 0
+
+INSERT INTO test1 VALUES (DEFAULT);
+
+Error: 1004(InvalidArguments), Columns and values number mismatch, columns: 3, values: 1
+
+INSERT INTO test1 VALUES (DEFAULT, DEFAULT, DEFAULT);
+
+Error: 1004(InvalidArguments), No valid default value can be build automatically, column: j
```
Comment (lizhemingi) [inline tests/cases/standalone/insert/insert_default.result:11]: Should be `can be built automatically` here ? It's a typo I think.

Could you please also fix the typo in code. It's my bad, sorry about that.

Comment (killme2008) [reply tests/cases/standalone/insert/insert_default.result:11]: `build` -> `built`.

Comment (waynexia) [reply tests/cases/standalone/insert/insert_default.result:11]: haha nice catch

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**